# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/udev/sbindir.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- systemd-258.2/rules.d/64-btrfs.rules.in.vanilla	2025-11-11 09:38:43.887958264 +0100
+++ systemd-258.2/rules.d/64-btrfs.rules.in	2025-11-11 09:38:52.987140201 +0100
@@ -12,6 +12,6 @@
 ENV{ID_BTRFS_READY}=="0", ENV{SYSTEMD_READY}="0"
 
 # reconsider pending devices in case when multidevice volume awaits
-ENV{ID_BTRFS_READY}=="1", RUN+="{{BINDIR}}/udevadm trigger -s block -p ID_BTRFS_READY=0"
+ENV{ID_BTRFS_READY}=="1", RUN+="{{SBINDIR}}/udevadm trigger -s block -p ID_BTRFS_READY=0"
 
 LABEL="btrfs_end"
--- systemd-258.2/rules.d/71-seat.rules.in.vanilla	2025-11-11 09:40:03.332083839 +0100
+++ systemd-258.2/rules.d/71-seat.rules.in	2025-11-11 09:40:12.184557885 +0100
@@ -70,7 +70,7 @@
 SUBSYSTEM=="usb", ATTR{idVendor}=="17e9", ATTR{idProduct}=="401a", ATTR{product}=="mimo inc", \
                   ATTR{../idVendor}=="058f", ATTR{../idProduct}=="6254", \
                   ENV{ID_AVOID_LOOP}=="", \
-                  RUN+="{{BINDIR}}/udevadm trigger --parent-match=%p/.."
+                  RUN+="{{SBINDIR}}/udevadm trigger --parent-match=%p/.."
 
 TAG=="seat", ENV{ID_PATH}=="", IMPORT{builtin}="path_id"
 TAG=="seat", ENV{ID_FOR_SEAT}=="", ENV{ID_PATH_TAG}!="", ENV{ID_FOR_SEAT}="$env{SUBSYSTEM}-$env{ID_PATH_TAG}"
--- systemd-258.2/meson.build.vanilla	2025-11-11 09:38:08.640962260 +0100
+++ systemd-258.2/meson.build	2025-11-11 09:38:33.772365886 +0100
@@ -271,6 +271,7 @@
 conf.set_quoted('PREFIX_NOSLASH',                             prefixdir_noslash)
 conf.set_quoted('RANDOM_SEED',                                randomseeddir / 'random-seed')
 conf.set_quoted('RANDOM_SEED_DIR',                            randomseeddir)
+conf.set_quoted('SBINDIR',                                    sbindir)
 conf.set_quoted('SSHCONFDIR',                                 sshconfdir)
 conf.set_quoted('SSHDCONFDIR',                                sshdconfdir)
 conf.set_quoted('SHELLPROFILEDIR',                            shellprofiledir)
